src/IR/ClassInstantiation.hs

```diff
@@ -1,15 +1,39 @@
 module IR.ClassInstantiation where
```

Comment:
You should indicate that this is currently a nop/throw an error if you find any classes or instances.